### PR TITLE
feat(components): Add `TranslateFromLeft` animation.

### PR DESCRIPTION
**Pull request by javieri-empathy**
EX-4772

# Description

Just adds a missing simple translate animation

### CODE REVIEW

File: packages/x-components/src/components/result/base-result-image.vue [hunk 1]
```diff
@@ -180,45 +191,47 @@
   .x-result-picture {
     min-width: 1px;
     min-height: 1px;
+    position: relative;
+
     &__image {
       max-width: 100%;
       max-height: 100%;
-      object-fit: fill;
+      object-fit: cover;
```
Comment (tajespasarela) [inline packages/x-components/src/components/result/base-result-image.vue:199]: change this by `contain`

File: packages/x-components/src/components/result/base-result-image.vue [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
+        v-if="imageSrc"
+        v-show="hasImageLoaded"
+        @error="flagImageAsFailed"
+        @load="flagImageLoaded"
+        :alt="result.name"
+        :src="imageSrc"
+        class="x-picture__image x-result-picture__image"
+        data-test="result-picture__image"
+      />
+      <NoElement v-else-if="hasAllImagesFailed" class="x-picture__image x-picture__image--fallback">
```
Comment (tajespasarela) [inline packages/x-components/src/components/result/base-result-image.vue:23]: Why is this a `NoComponent`? can't be a `<template>`?

**Review by tajespasarela — CHANGES_REQUESTED** [review]

File: packages/x-components/src/components/result/base-result-image.vue (hunk 2 above)
Comment (javieri-empathy) [reply packages/x-components/src/components/result/base-result-image.vue:23]: Are you asking me 🤣? 
image

I remember discussing about using `NoElement` to add classes to slots, and I guess that's the case here, adding the design system classes to the slot content. But I'm not sure if there are other reasons.

Comment (tajespasarela) [reply packages/x-components/src/components/result/base-result-image.vue:23]: Don't play with my poor memory!

I guess it was for the classes as you say.

**Review by tajespasarela — APPROVED** [review]

**Pull request merged**